zulipterminal/cli/run.py

```diff
@@ -174,11 +174,14 @@ def main() -> None:
     except Exception as e:
         if args.debug:
             # A unexpected exception occurred, open the debugger in debug mode
-            import pudb
-            pudb.post_mortem()
 
-        sys.stdout.flush()
-        traceback.print_exc(file=sys.stderr)
+            sys.stdout.flush()
```

Comment:
Should this line go with the spinning cursor code?

Reply:
You're going to address in a future version?

Reply:
I think we should leave it there to print whatever the message is in the buffer before printing traceback?